mmpose/visualization/fast_visualizer.py

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import cv2
+
+
+class FastVisualizer:
```

Comment:
It would be better if we added a unit test for `FastVisualizer`

Reply:
You're right. I'll add a ut later.